Rakefile.rb

```diff
@@ -12,20 +13,26 @@
 
   desc 'Pull the integrations repo'
   task :'pull-integrations' do
-    sh "cd /#{ENV['INTEGRATIONS_REPO']} &&
-        git fetch --all &&
-        git checkout dd-check-#{ENV['INTEGRATION']}-#{ENV['VERSION']} &&
-        git reset --hard"
+    integration_branch = ENV['VERSION'] || 'master'
+
+    sh "git -C /#{ENV['INTEGRATIONS_REPO']} fetch --all"
```

Comment:
Do you think this is a better solution than `-C`? I am find handing this off to git rather than dealing with it in Ruby.